x-pack/plugins/cloud_security_posture/server/tasks/findings_stats_task.ts

```diff
@@ -94,7 +97,11 @@ export function taskRunner(coreStartServices: CspServerPluginStartServices, logg
         try {
           logger.info(`Runs task: ${CSPM_FINDINGS_STATS_TASK_TYPE}`);
           const esClient = (await coreStartServices)[0].elasticsearch.client.asInternalUser;
-          const status = await aggregateLatestFindings(esClient, logger);
+          const encryptedSoClient = (
```

Comment:
the `await coreStartServices` call is duplicated, what about moving `(await coreStartServices)[0]` to own constant and use the result for both `esClient` and `encryptedSoClient` ?